### PR TITLE
RXR-1050: Incorporate docs from PKPDsim-docs

### PR DESCRIPTION
**Pull request by karawoo**
Incorporates the content from PKPDsim-docs as vignettes within PKPDsim. Also converts them to .Rmd files so that the code can be run and results shown.

Some of the pages use packages that aren't on CRAN, or that aren't dependencies of PKPDsim. I removed the pages on plotting and shiny that use PKPDplot and PKPDshiny. For vignettes that use tidyverse, ggplot2, etc. I have added these to .Rbuildignore: they will not be included with the CRAN version of PKPDsim, but they _will_ still appear on the pkgdown site (which I think is more useful/visible anyway).

This is a draft PR as I still need to check some of the results and make sure they look ok. But @roninsightrx I could use your help on a couple of pieces. I will add in-line comments below.

### CODE REVIEW

File: vignettes/model.Rmd [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+### Dosing / bioavailability
+
+The default dosing compartment and bioavailability can be specified using the `dose` argument. By default, the dose will go into compartment `1`, with a bioavailability of `1`. The `bioav` element in the list can be either a number or a character string referring a parameter.
+
+```{r bioav, eval=FALSE}
+pk1 <- new_ode_model(code = "
+  dAdt[1] = -KA * A[1]
+  dAdt[2] = +KA * A[1] -(CL/V) * A[2]
+",
+dose = list(cmt = 1, bioav = "F1"))
```
Comment (karawoo) [inline vignettes/model.Rmd:101]: This line throws an error. Should it just be `bioav = "1"`?

File: vignettes/output.Rmd [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  mod_1cmt_iv,
+  parameters = p,
+  regimen = reg,
+  covariates_table = cov_table,
+  covariates_implementation = list(SCR = "interpolate"),
+  omega = NULL,
+  n_ind = 3,
+  only_obs = T,
+  output_include = list(parameters = TRUE, covariates=TRUE)
+)
```
Comment (karawoo) [inline vignettes/output.Rmd:70]: This vignette originally had an incomplete example that didn't define all the inputs to `sim()`, so I copied this example from the tests.

File: vignettes/estimation-nlmixr.Rmd [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  parameters = par,
+  omega = c(0.1, 0.05, 0.1),
+  res_var = list(prop = 0.1, add = 0.1),
+  log_transform = T)
+
+## Perform fit using nlmixr
+fit <- nlmixr(object = f,
+       data = d,
+       est = "saem",
+       saemControl(n.burn = 100, n.em = 200))
```
Comment (karawoo) [inline vignettes/estimation-nlmixr.Rmd:84]: This throws an error for me locally

**Comment by roninsightrx** [comment]
Went through all vignettes and made updates: https://github.com/InsightRX/PKPDsim/pull/47.

**Review by roninsightrx — APPROVED** [review]
nice, works

**Pull request merged**